src/utils/clack-utils.ts

```diff
@@ -715,7 +728,7 @@ export async function runPrettierIfInstalled(): Promise<void> {
       });
     } catch (e) {
       prettierSpinner.stop('Prettier failed to run.');
-      clack.log.error(
+      clack.log.warn(
```

Comment:
I think it's ok to downgrade this to a warning because it won't (shouldn't) cause the wizard to error. It'll just continue with the next step.